OPHD/UI/PopulationPanel.cpp

```diff
@@ -112,33 +116,53 @@ void PopulationPanel::update()
 
 	auto position = NAS2D::Point{positionX() + constants::Margin, positionY() + constants::Margin};
 
-	// POPULATION
+	// POPULATION Statistics
 	renderer.drawText(mFontBold, constants::PopulationBreakdown, position);
 	const auto& population = mPopulation->getPopulations();
 	const std::array populationData
 	{
-		std::tuple{NAS2D::Rectangle<int>{{0, 96}, {IconSize, IconSize}}, population.child, std::string("Children")},
-		std::tuple{NAS2D::Rectangle<int>{{32, 96}, {IconSize, IconSize}}, population.student, std::string("Students")},
-		std::tuple{NAS2D::Rectangle<int>{{64, 96}, {IconSize, IconSize}}, population.worker, std::string("Workers")},
-		std::tuple{NAS2D::Rectangle<int>{{96, 96}, {IconSize, IconSize}}, population.scientist, std::string("Scientists")},
-		std::tuple{NAS2D::Rectangle<int>{{128, 96}, {IconSize, IconSize}}, population.retiree, std::string("Retired")},
+		std::tuple{NAS2D::Rectangle<int>{{0, 96},	{IconSize, IconSize}},	population.child, std::string("Children")},
+		std::tuple{NAS2D::Rectangle<int>{{32, 96},	{IconSize, IconSize}},	population.student, std::string("Students")},
+		std::tuple{NAS2D::Rectangle<int>{{64, 96},	{IconSize, IconSize}},	population.worker, std::string("Workers")},
+		std::tuple{NAS2D::Rectangle<int>{{96, 96},	{IconSize, IconSize}},	population.scientist, std::string("Scientists")},
+		std::tuple{NAS2D::Rectangle<int>{{128, 96},	{IconSize, IconSize}},	population.retiree, std::string("Retired")},
 	};
 
 	position.y += fontBoldHeight + constants::Margin;
 	const auto textOffset = NAS2D::Vector{IconSize + constants::Margin, (IconSize / 2) - (fontHeight / 2)};
+	const int populationStatisticsWidth = mFont.width(constants::PopulationBreakdown) + constants::Margin;
+
 	for (const auto& [imageRect, personCount, personRole] : populationData)
 	{
 		renderer.drawSubImage(mIcons, position, imageRect);
 
-		const auto roleCount = std::to_string(personCount);
 		renderer.drawText(mFont, personRole + ": ", position + textOffset);
 
-		const NAS2D::Point<int> labelPosition = {positionX() + mPopulationPanelWidth - mFont.width(roleCount) - constants::Margin , position.y + textOffset.y};
-		renderer.drawText(mFont, roleCount, labelPosition);
+		const NAS2D::Point<int> labelPosition = {positionX() + mPopulationPanelWidth - mFont.width(std::to_string(personCount)) - constants::Margin, position.y + textOffset.y};
+		renderer.drawText(mFont, std::to_string(personCount), labelPosition);
+
```

Comment:
This makes multiple calls to `std::to_string` with the same argument. We should probably cache the result to avoid having to call it twice. Notably, the `std::to_string` function will likely cause memory allocation (assuming no short string optimization), and we should probably try to avoid multiple short allocations when reasonable to avoid them.

It shouldn't cost anything to extend the lifetime of a temporary to the body of the loop.